apps/server/src/infra/etherpad-client/etherpad-client.adapter.ts

```diff
@@ -50,6 +52,29 @@ export class EtherpadClientAdapter {
 		}
 	}
 
+	public async listPadsOfAuthor(userId: EntityId): Promise<PadId[]> {
+		const response = await this.tryGetPadsOfAuthor(userId);
+		const pads = this.handleEtherpadResponse<InlineResponse2002>(response, { userId });
+
+		if (!this.isObject(pads)) {
+			throw new InternalServerErrorException('Etherpad listPadsOfAuthor response is not an object');
+		}
+
+		const padIds = pads?.padIDs as PadId[];
+
+		return padIds;
+	}
+
+	private async tryGetPadsOfAuthor(userId: string): Promise<AxiosResponse<InlineResponse2002>> {
```

Comment:
Incorrect parameter that expects a authorId

Reply:
fixed